app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java

```diff
@@ -112,18 +112,18 @@ public static OkHttpClient getClient() {
     }
 
     public static OkHttpClient createClient() {
-        OkHttpClient client = null;
+        OkHttpClient.Builder b = new OkHttpClient.Builder()
+                .readTimeout(45, TimeUnit.SECONDS);
```

Comment:
Yeah this one is increased specifically to allow the download of big pdf files (which can take a lot of time to generate on the server's side). I actually don't know about the others, but I definitely want this as an advanced setting (well, when https://github.com/wallabag/android-app/issues/143 is done).